tests/Runner/RunnerTest.php

```diff
@@ -184,7 +184,7 @@ public function testThatParallelFixOfInvalidFileReportsToErrorManager(): void
     /**
      * @requires OS Darwin|Windows
      *
-     * @TODO Fix it for Linux - somehow this test reaches timeout in CI, even though it works on Linux through Docker.
+     * @TODO v4 Fix it for Linux - somehow this test reaches timeout in CI, even though it works on Linux through Docker. @Wirone
```

Comment:
I don't think we need to wait for v4 to fix it. I have a suspicion what's wrong here - probably it's a race condition between spawning child processes and stopping all of them after encountering violation. This error looks like 1st process fails on violation and kills all processes before 2nd process' `hello` action is handled, so `$processPool->getProcess()` fails with exception. Maybe timeout is somehow similar, but 2nd process is stopped after process is fetched from the pool, but connection can't be established? I would like to look at this, but I don't consider it as a blocker.

Reply:
> I don't think we need to wait for v4 to fix it

my point was not to wait for v4 indeed, but to not release v4 with parallel execution by default if this is not yet solved.

happy to have a v4 mentioned again, with context of above.
I use `grep TODO v4` as quick filter.